artifacts/rhdh-plugins-reference/argocd/argocd-plugin-readme.adoc

```diff
@@ -1,322 +1,114 @@
+.Procedure
 
-. To create multiple components that fetch data from different ArgoCD instances, add a proxy configuration for each instance as shown in the following example:
+. Add the following to your dynamic-plugins ConfigMap to enable the Argo CD plugin.
 +
---
 [source,yaml]
 ----
-proxy:
-  ...
-
-  '/argocd/api':
-    target: https://<someAddress>/api/v1/
-    changeOrigin: true
-    secure: false
-    headers:
-      Cookie:
-        $env: ARGOCD_AUTH_TOKEN
-
-  '/argocd/api2':
-    target: https://<otherAddress>/api/v1/
-    changeOrigin: true
-    secure: false
-    headers:
-      Cookie:
-        $env: ARGOCD_AUTH_TOKEN2
-
+global:
+  dynamic:
+    includes:
+      - dynamic-plugins.default.yaml
+    plugins:
+      - package: ./dynamic-plugins/dist/roadiehq-backstage-plugin-argo-cd-backend-dynamic
+        disabled: false
+      - package: ./dynamic-plugins/dist/janus-idp-backstage-plugin-argocd
+        disabled: false
 ----
---
 
-. Add the required authentication tokens to environment variables, such as `ARGOCD_AUTH_TOKEN2`.
-+
---
-After adding the authentication tokens, add the URL to the desired proxy path in the following component definition annotations:
+== For Users
 
-`argocd/proxy-url: '/argocd/api2'`
+.Prerequisites
 
-The `argocd/proxy-url` annotation defaults to `'/argocd/api'`. Therefore, `argocd/proxy-url` is not required if there is a single proxy configuration.
---
+* You have enabled the Argo CD plugin in {product} {product-very-short}.
 
-.Option 2: Using the ArgoCD backend plugin
+.Procedures
 
-. To create multiple components that fetch data from different ArgoCD instances, set the ArgoCD instance URL dynamically by adding the following configuration to `app-config.yaml` file:
-+
---
-[source,yaml]
-----
-argocd:
-  username: ${ARGOCD_USERNAME}
-  password: ${ARGOCD_PASSWORD}
-  waitCycles: 25
-  appLocatorMethods:
-    - type: 'config'
-      instances:
-        - name: argoInstance1
-          url: https://argoInstance1.com
-          token: ${ARGOCD_AUTH_TOKEN} # optional
-        - name: argoInstance2
-          url: https://argoInstance2.com
-          # dedicated username/password for this instance
-          username: ${ARGOCD_USERNAME_INSTANCE_2} # optional
-          password: ${ARGOCD_PASSWORD_INSTANCE_2} # optional
+. On the {product-very-short} platform, navigate to the *Catalog* and select an appropriate component.
 
-----
-
-A loop is created between deleting ArgoCD project and application to check the deletion of application occurring before the deletion of the project. If waitCycles is set to 25, then the loop can last for 75 seconds before erroring out.
-
-With the previous configuration, the ArgoCD plugin fetches the ArgoCD instances an application is deployed to and use the ArgoCD backend plugin (`backstage-plugin-argo-cd-backend`) to reach out to each ArgoCD instance based on the following mapping:
+. Select *CD* tab to view insights into deployments managed by Argo CD. 
```

Comment:
Click the **CD** tab

Reply:
@linfraze Thank you for your suggestion. I decided to use "Select" based on this:
![image](https://github.com/user-attachments/assets/920f94c6-8748-4d0e-a304-9ee07b7dec77)

Let me know if you disagree or have some other references.
Thanks again.

Reply:
Based on this discussion - https://redhat-internal.slack.com/archives/C04P5S34GP2/p1722440235254279
...going with Select.

Reply:
@gtrivedi88 that's fine. I appreciate you starting the discussion in the CCS channel. We all learn a lot that way.